sdk/python/src/dagger/client/gen.py

```diff
@@ -4158,13 +4477,11 @@ def directory(self, *, id: DirectoryID | None = None) -> Directory:
 
     @typecheck
     def file(self, id: FileID) -> File:
-        """Loads a file by ID.
-
-        .. deprecated::
-            Use :py:meth:`load_file_from_id` instead.
+        """.. deprecated::
+        Use loadFileFromID instead.
         """
         warnings.warn(
-            'Method "file" is deprecated: Use "load_file_from_id" instead.',
+            'Method "file" is deprecated: Use loadFileFromID instead.',
```

Comment:
We have an undocumented convention since the beginning to use backticks in fields referenced in deprecation messages so that SDKs can rename the field to its conventions.

Reply:
Ahh thanks, I had a vague memory there but couldn't remember if it was load bearing, makes a lot of sense. Will fix!

Reply:
fixed in https://github.com/dagger/dagger/pull/6422: https://github.com/dagger/dagger/pull/6422/commits/524c3c3a345ee100d45a56f4e204ae03fad78609